webapp/src/lib/i18n/index.ts

```diff
@@ -0,0 +1,10 @@
+import { createI18n } from "@inlang/paraglide-js-adapter-sveltekit"
+
+import * as m from '$paraglide/messages'
+import * as runtime from "$paraglide/runtime"
+
+// 
+
+export const i18n = createI18n(runtime)
```

Comment:
you can add several parameters to this function. You can add translations for the routes and I think it is quite useful for better SEO in public routes. 
for example:
```
export const i18n = createI18n(runtime, {
	pathnames: {
		 '/my': {
		 	en: m.my(undefined, {{languageTag:"en"} ),
			it: m.my(undefined, {{languageTag:"it"} )
		 },
		 etc..
})
```

Reply:
Can we have a single source of truth for the list of the supported languages?
Is this the right place?

Reply:
Is quite ugly, but I tested it and it works (as long you add the translations of the routes in to the messages):
```
type Routes = `/${string}`;

const translatedRoutes = ['home'];

const pathnames: Record<Routes, Record<string, Routes>> = translatedRoutes.reduce(
	(acc: Record<Routes, Record<string, Routes>>, route) => {
		acc[`/${route}`] = runtime.availableLanguageTags.reduce(
			(langs: Record<string, Routes>, tag) => {
				// @ts-expect-error because every translated route it should be added to messages
				langs[tag] = `/${m[route](undefined, { languageTag: tag }) || route}` as Routes;
				return langs;
			},
			{}
		);
		return acc;
	},
	{}
);
```

Reply:
Given that almost all routes in signroom are private, I don't think we need to do this actually